R/data-process.R

```diff
@@ -211,7 +211,7 @@ gen_data_weighted_ds <- function(data_weighted) {
           min = ~ min(.x, na.rm = TRUE),
           max = ~ max(.x, na.rm = TRUE)
         ),
-        .names = "{stringr::str_sub(.col, 4)}_{.fn}"
+        .names = "{.col}_{.fn}"
```

Comment:
Where will the names come from? Is it configuration?

Reply:
The `.names` argument helps name the output columns. 
The `{.col}` represents the particular column (yes, from the config file) and `{.fn}` is the name of the function applied (for example, mean, min, max). 
So finally, the output column name will be connecting `{.col}` and the corresponding `{.fn}` using an underscore `_`.